crates/http/src/wagi.rs

```diff
@@ -143,7 +143,14 @@ impl HttpExecutor for WagiHttpExecutor {
             .or_else(ignore_successful_proc_exit_trap)?;
         tracing::info!("Module execution complete");
 
-        util::compose_response(&stdout_buffer.take())
+        let stdout = stdout_buffer.take();
+        ensure!(
+            !stdout.is_empty(),
+            "The {component:?} component is configured to use the WAGI executor \
```

Comment:
super nit: I think `component` is `&str` at this point so doesn't need the debug formatter

Reply:
I like the debug formatter for error messages and logging because it quotes and escapes which can reveal surprise characters.